--- configure.in.orig	2015-12-08 19:48:20.706653764 +0100
+++ configure.in	2015-12-08 19:50:39.586652933 +0100
@@ -288,12 +288,12 @@
     RTNAME=rtai-3.0
     RTPREFIX=rtai
     RTAI=3
-    RTDIR=`$RTS --prefix`
+    RTDIR=`$RTS --prefix -o`
     RTDIR=$(cd $RTDIR ; pwd -P )
-    RTFLAGS=`$RTS --module-cflags`
+    RTFLAGS=`$RTS --module-cflags -o`
     RTFLAGS="$RTFLAGS -DRTAI=3"
-    KERNELDIR=`$RTS --linux-dir`
-    RTARCH=`$RTS --arch`
+    KERNELDIR=`$RTS --linux-dir -o`
+    RTARCH=`$RTS --arch -o`
 ;;
 uspace)
     RTPREFIX=uspace
@@ -357,7 +357,7 @@
 AC_MSG_CHECKING([for cc version])
 
 if (test ${RTS##*/} = rtai-config ) ; then
-  RTSCC=`$RTS --cc`
+  RTSCC=`$RTS --cc -o`
   AC_MSG_RESULT(found $RTSCC in rtai-config)
 else
   AC_MSG_RESULT(not specified)
@@ -518,7 +518,7 @@
 
 # Do a quick test for module extension..
 AC_MSG_CHECKING([for kernel module extension])
-MODEXT=`$RTS --module-ext 2>/dev/null` || MODEXT=".o"
+MODEXT=`$RTS --module-ext -o 2>/dev/null` || MODEXT=".o"
 if test "x" = "x$MODEXT"; then
     MODEXT=".o"
 fi
@@ -589,7 +589,7 @@
 # if not specified, ask the realtime config script where the directory is
 # if the config script returns an error, set MODULE_DIR to nothing
 if test "x" = "x$MODULE_DIR" ; then
-  MODULE_DIR=`$RTS --module-dir 2>/dev/null`
+  MODULE_DIR=`$RTS --module-dir -o 2>/dev/null`
   if ! test -d $MODULE_DIR ; then
     AC_MSG_ERROR([$RTS --module-dir did not specify an existing directory.  You must use specify it using --with-module-dir=...])
   fi
