diff -ruN a/src/emc/iotask/Submakefile b/src/emc/iotask/Submakefile
--- a/src/emc/iotask/Submakefile	2020-02-26 04:36:32.995394787 -0500
+++ b/src/emc/iotask/Submakefile	2020-02-26 04:44:14.768165424 -0500
@@ -4,11 +4,11 @@
 
 ../bin/io: $(call TOOBJS, $(IOSRCS)) ../lib/liblinuxcnc.a ../lib/libnml.so.0 ../lib/liblinuxcnchal.so.0 ../lib/liblinuxcncini.so.0
 	$(ECHO) Linking $(notdir $@)
-	@$(CXX) $(LDFLAGS) -o $@ $^ 
+	@$(CXX) $(LDFLAGS) -o $@ $^ -ltirpc
 
 ../bin/iov2: $(call TOOBJS, $(IOV2SRCS)) ../lib/liblinuxcnc.a ../lib/libnml.so.0 ../lib/liblinuxcnchal.so.0 ../lib/liblinuxcncini.so.0
 	$(ECHO) Linking $(notdir $@)
-	@$(CXX) $(LDFLAGS) -o $@ $^
+	@$(CXX) $(LDFLAGS) -o $@ $^ -ltirpc
 
 TARGETS += ../bin/io ../bin/iov2
 
diff -ruN a/src/emc/sai/Submakefile b/src/emc/sai/Submakefile
--- a/src/emc/sai/Submakefile	2020-02-26 04:36:33.005394414 -0500
+++ b/src/emc/sai/Submakefile	2020-02-26 04:44:31.217551216 -0500
@@ -25,4 +25,4 @@
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
 	$(Q)$(CXX) $(LDFLAGS) $(PROFILE_LDFLAGS) \
-	-o $@ $^ $(ULFLAGS) -l$(BOOST_PYTHON_LIB) $(PYTHON_LIBS) $(LIBREADLINE)
+	-o $@ $^ $(ULFLAGS) -l$(BOOST_PYTHON_LIB) $(PYTHON_LIBS) $(LIBREADLINE) -ltirpc
diff -ruN a/src/emc/task/Submakefile b/src/emc/task/Submakefile
--- a/src/emc/task/Submakefile	2020-02-26 04:36:33.005394414 -0500
+++ b/src/emc/task/Submakefile	2020-02-26 04:43:51.409037590 -0500
@@ -9,7 +9,7 @@
 	../lib/liblinuxcncini.so.0 \
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
-	@$(CXX) $(LDFLAGS) -o $@ $^ 
+	@$(CXX) $(LDFLAGS) -o $@ $^ -ltirpc
 TARGETS += ../bin/linuxcncsvr
 
 # disabled:	emc/task/iotaskintf.cc
@@ -42,5 +42,5 @@
 	../lib/libpyplugin.so.0 \
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
-	$(Q)$(CXX) -o $@ $^ $(LDFLAGS) -l$(BOOST_PYTHON_LIB) $(PYTHON_LIBS)
+	$(Q)$(CXX) -o $@ $^ $(LDFLAGS) -l$(BOOST_PYTHON_LIB) $(PYTHON_LIBS) -ltirpc
 TARGETS += ../bin/milltask
diff -ruN a/src/emc/usr_intf/Submakefile b/src/emc/usr_intf/Submakefile
--- a/src/emc/usr_intf/Submakefile	2020-02-26 04:36:33.015394041 -0500
+++ b/src/emc/usr_intf/Submakefile	2020-02-26 04:43:20.860178098 -0500
@@ -60,7 +60,7 @@
 	../lib/liblinuxcncini.so.0 \
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
-	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ -lpthread
+	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ -lpthread -ltirpc
 TARGETS += ../bin/linuxcncrsh
 
 ../bin/schedrmt: $(call TOOBJS, $(EMCSCHEDSRCS)) \
@@ -69,7 +69,7 @@
 	../lib/liblinuxcncini.so.0 \
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
-	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ -lpthread
+	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ -lpthread -ltirpc
 TARGETS += ../bin/schedrmt
 
 ../bin/linuxcnclcd: $(call TOOBJS, $(EMCLCDSRCS)) \
@@ -78,7 +78,7 @@
 	../lib/liblinuxcncini.so.0 \
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
-	$(Q)$(CXX) $(LDFLAGS) -o $@ $^
+	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ -ltirpc
 TARGETS += ../bin/linuxcnclcd
 
 
@@ -89,19 +89,19 @@
 	../lib/liblinuxcnchal.so.0 \
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
-	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ 
+	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ -ltirpc
 TARGETS += ../bin/halui
 
-ifeq "$(HAVE_NCURSES)" "yes"
-../bin/keystick: $(call TOOBJS, $(KEYSTICKSRCS)) \
-	../lib/liblinuxcnc.a \
-	../lib/libnml.so.0 \
-	../lib/liblinuxcncini.so.0 \
-	../lib/librtapi_math.so.0
-	$(ECHO) Linking $(notdir $@)
-	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ $(KEYSTICKLIBS)
-TARGETS += ../bin/keystick
-endif
+#ifeq "$(HAVE_NCURSES)" "yes"
+#../bin/keystick: $(call TOOBJS, $(KEYSTICKSRCS)) \
+#	../lib/liblinuxcnc.a \
+#	../lib/libnml.so.0 \
+#	../lib/liblinuxcncini.so.0 \
+#	../lib/librtapi_math.so.0
+#	$(ECHO) Linking $(notdir $@)
+#	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ $(KEYSTICKLIBS)
+#TARGETS += ../bin/keystick
+#endif
 
 ifeq "$(HAVE_XAW)" "yes"
 $(call TOOBJSDEPS,$(XEMCSRCS)): EXTRAFLAGS = $(CFLAGS_X)
@@ -111,7 +111,7 @@
 	../lib/liblinuxcncini.so.0 \
 	../lib/librtapi_math.so.0
 	$(ECHO) Linking $(notdir $@)
-	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ $(XLIBS)
+	$(Q)$(CXX) $(LDFLAGS) -o $@ $^ $(XLIBS) -ltirpc
 TARGETS += ../bin/xlinuxcnc
 endif
 
@@ -124,6 +124,6 @@
 	$(ECHO) Linking $(notdir $@)
 	$(Q)$(CXX) $(LDFLAGS) $(BOOST_LDFLAGS) -o $@ $(ULFLAGS) $^ \
 	    $(BOOST_THREAD_LIB) $(BOOST_SYSTEM_LIB) \
-	    -lpthread -ldl	
+	    -lpthread -ldl -ltirpc
 TARGETS += ../bin/emcweb
 endif
