--- base/sched/GNUmakefile.am.old	2016-03-04 12:28:27.883176827 +0000
+++ base/sched/GNUmakefile.am	2016-03-04 12:28:55.363176664 +0000
@@ -44,7 +44,6 @@
 	@RTAI_KBUILD_TOP@ \
 	@RTAI_KBUILD_CMD@ rtai_extradef="$(rtai_sched_extradef)" \
 	@RTAI_KBUILD_BOTTOM@
-	${SHELL} $(top_srcdir)/base/sched/selfcalibrate.sh $(rtai_target_arch)
 	@(touch calibration_placeholder.c)
 
 rtai_sched$(modext): @RTAI_KBUILD_ENV@
@@ -115,4 +114,4 @@
 # liblxrt is always built
 SUBDIRS = liblxrt
 
-EXTRA_DIST = Makefile.kbuild selfcalibrate.sh
+EXTRA_DIST = Makefile.kbuild
