--- src/configure.ac.orig	2020-05-03 22:29:19.397713147 -0400
+++ src/configure.ac	2020-05-03 22:29:28.817365543 -0400
@@ -2061,7 +2061,7 @@
 	AC_MSG_ERROR([Python missing.  Install it or specify --disable-python to skip the parts of Machinekit that depend on Python])
     fi
 
-    AX_PYTHON_DEVEL(>= 2.6)
+    AX_PYTHON_DEVEL(>= '2.6')
     AC_PRUNE_DEFAULT_LDFLAGS(PYTHON_LIBS)
 
 #    AC_MSG_CHECKING([match between tk and Tkinter versions])
@@ -1828,7 +1828,7 @@
     AX_BOOST_BASE(1.34, , [AC_MSG_ERROR([Boost 1.34.0 or newer required])])
 
     AC_MSG_CHECKING(for site-package location)
-    SITEPY=`$PYTHON -c 'import distutils.sysconfig; print distutils.sysconfig.get_python_lib(1)'`
+    SITEPY=`$PYTHON -c 'import distutils.sysconfig; print(distutils.sysconfig.get_python_lib(1))'`
     AC_MSG_RESULT($SITEPY)
 
     AC_CHECK_HEADERS(GL/gl.h GL/glu.h,[],[AC_MSG_ERROR([Required OpenGL header missing.  Install it, or specify --disable-python to skip the parts of Machinekit that depend on Python])])
