--- src/machinetalk/nanopb/generator/nanopb_generator.py	2021-07-10 10:11:39.000000000 -0400
+++ src/machinetalk/nanopb/generator/nanopb_generator.py	2021-07-10 10:11:52.000000000 -0400
@@ -1289,7 +1289,7 @@
         if options.verbose:
             sys.stderr.write('Reading options from ' + optfilename + '\n')
 
-        Globals.separate_options = read_options_file(open(optfilename, "rU"))
+        Globals.separate_options = read_options_file(open(optfilename, "r"))
     else:
         # If we are given a full filename and it does not exist, give an error.
         # However, don't give error when we automatically look for .options file
