--- src/machinetalk/nanopb/generator/nanopb_generator.py	2021-07-10 09:37:48.000000000 -0400
+++ src/machinetalk/nanopb/generator/nanopb_generator.py	2021-07-10 09:48:59.000000000 -0400
@@ -1367,7 +1367,7 @@
         msvcrt.setmode(sys.stdin.fileno(), os.O_BINARY)
         msvcrt.setmode(sys.stdout.fileno(), os.O_BINARY)
     
-    data = sys.stdin.read()
+    data = sys.stdin.buffer.read()
     request = plugin_pb2.CodeGeneratorRequest.FromString(data)
     
     import shlex
